portfolio/src/main/webapp/script.js

```diff
@@ -69,3 +72,12 @@ function scrollToElement(elementId, duration = 300)
                     scrollTop: $('#' + elementId).offset().top - navOffset
                 }, duration);
 }
+
+/** 
+ * Function that loads greeting-container content
+ */
+function LoadGreeting() {
```

Comment:
This function seems short enough to have the functionality inlined where it is actually called. It would make sense as a function if it would be called in different places, or if you parameterize it. In the current state you can simply move the body of the function to the init(). If you see that there is a way to parameterize it in the future or move common functionality from different places in one function you can refactoring it, but this is not needed right now.